arch/arm/src/stm32h7/hardware/stm32_fdcan.h

```diff
@@ -0,0 +1,1594 @@
+/****************************************************************************
+ * arch/arm/src/stm32h7/hardware/stm32_fdcan.h
+ *
+ *   Copyright (C) 2020 Jacob Crabill. All rights reserved.
+ *   Author: Jacob Crabill <jacob@volansi.com>
```

Comment:
I think we should use Apache Software Foundation license header

Reply:
Ah, missed that.  Will update.